src/components/Navbar.tsx

```diff
@@ -21,13 +23,14 @@ const Navbar = () => {
       {item}
     </Link>
   ))
+
   return (
     <div className="mx-auto hidden w-full max-w-7xl flex-shrink-0 items-center justify-between pt-8 font-geologica md:flex">
       <Link to={'/'} className="flex cursor-pointer flex-col items-center justify-center pl-3 text-app-blue">
         <Logo />
         <p className="uppercase">tetrahedron</p>
       </Link>
-      <div className="flex items-center justify-center gap-7">{menuItems}</div>
+      <div className="flex items-center justify-center gap-7">{displayMenuTitles ? menuItems : null}</div>
```

Comment:
Do we need the empty div element when we don't show menu?
Maybe let's do like this:
`{displayMenuTitles && <div className="flex items-center justify-center gap-7">{menuItems}</div>}`